pkg/simple/client/admission/options.go

```diff
@@ -0,0 +1,30 @@
+package admission
+
+import (
+	"github.com/spf13/pflag"
+	"kubesphere.io/kubesphere/pkg/utils/reflectutils"
+)
+
+type Options struct {
+	Enable                   bool `json:"enable" yaml:"enable"`
+	EnableGatekeeperProvider bool `json:"enable_gatekeeper_provider" yaml:"enable_gatekeeper_provider"`
```

Comment:
```suggestion
	EnableGatekeeperProvider bool `json:"enableGatekeeperProvider" yaml:"enableGatekeeperProvider"`
```

We'd better use a lower camel case if used in the YAML configuration.